cmd/aperturectl/cmd/utils/controller.go

```diff
@@ -115,6 +153,33 @@ func (c *ControllerConn) PreRunE(_ *cobra.Command, _ []string) error {
 		if err != nil {
 			return err
 		}
+	} else if c.apiKey == "" && !c.kube {
```

Comment:
Should it be just `else if c.config != ""`?

Otherwise if someone has local controller without api keys (but not via --kube), we will try to load config.

Reply:
Made the changes. Thank you